objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestYoloProcessor.java

```diff
@@ -36,6 +36,8 @@
     static final String PB_FILE            = "yolo.pb";
     static final String OUTPUT_DIR         = "src/test/resources/out";
     static final int    SAVE_RATE          = 2; // Saves every other so that we can know it counts correctly
+    static final String AUTH_TOKEN         = "Ax6jSjrrwqnaHmZtWhXvVV1ym8ARfpta6wwmuPhy928=";
```

Comment:
This is a security issue since you're putting a token allowing folks to get into VANTIQ (as you) out in public.  We need to remove this token.

Better to allow it to be added as a gradle property or java property & picked up at runtime.  If not present, don't run that test.  A bit more complicated but needs to be done.